### PR TITLE
fix(mrc): set icon actionmenu compact ellipsisvertical

### LINKED ISSUE #14172: [manager-components]: ActionMenu design
### On which entity, you would like to report a bug?

ActionMenu in Datagrid (compact)

### Description

On the last version of the MRC with ODS v18, I've been told that the design of the ActionMenu button in a Datagrid row should be with variant `ghost` and `ellipsis-vertical` 

Actual design:

image

Should be (no border)

image

### Steps to reproduce

just browse [component in storybook](https://ovh.github.io/manager/storybook-static/index.html?path=/story/components-table--default-props)

### Browser

_No response_

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by tristanwagner**
ref: ISSUE-14172

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | fix #14172  
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by anooparveti — CHANGES_REQUESTED** [review]
Since, action menu is different when it's displayed on dashboard and listing page. This component should accept a new prop called 'mode' which can be with button or ghost and decide what to display.

File: packages/manager-react-components/src/components/navigation/menus/action/action.component.tsx [hunk 1]
```diff
@@ -23,11 +23,15 @@ export interface ActionMenuItem {
   className?: string;
 }
 
+export enum ActionMenuMode {
+  STANDARD = 'standard',
+  DATAGRID = 'datagrid',
+  CARD = 'card',
```
Comment (anooparveti) [inline packages/manager-react-components/src/components/navigation/menus/action/action.component.tsx:29]: This component need not know the parent components which are using it. So can we make these mode more generic? 
For example, this configuration will not be scalable if we want to use the same display as Datagrid in other component like form or so on.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager-react-components/src/components/navigation/menus/action/action.component.tsx (hunk 1 above)
Comment (aboungnaseng-ovhcloud) [reply packages/manager-react-components/src/components/navigation/menus/action/action.component.tsx:29]: Using the property 'variant' is not enough ? Or we want to be more restrictive ?

Comment (tristanwagner) [reply packages/manager-react-components/src/components/navigation/menus/action/action.component.tsx:29]: I think this component has 3 use cases and the current modes make it obvious what mode you need to choose for which use case, but standard, which I didn't find a better name

please provide me mode names if you have an idea

Comment (tristanwagner) [reply packages/manager-react-components/src/components/navigation/menus/action/action.component.tsx:29]: would "STANDARD", "COMPACT_OUTLINE" and "COMPACT_GHOST" be better ?

**Review by aboungnaseng-ovhcloud — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**